apps/editing-toolkit/package.json

```diff
@@ -79,7 +79,7 @@
 		"@wordpress/data": "^6.1.5",
 		"@wordpress/data-controls": "^2.2.8",
 		"@wordpress/date": "^4.2.3",
-		"@wordpress/dependency-extraction-webpack-plugin": "^3.2.1",
+		"@wordpress/dependency-extraction-webpack-plugin": "^3.4.1",
```

Comment:
This will get updated in https://github.com/Automattic/wp-calypso/pull/60161, though it shouldn't be a problem to update individually if everything passes here

Reply:
I'm surprised it _didn't_ end up getting updated. I guess the next round of updates will do it. Either way, this should be fine.

Reply:
Interestingly enough, the renovate PR didn't update the `@wordpress/dependency-extraction-webpack-plugin` to 3.4.1. I had to do it manually after merging `trunk` 🤔